app/src/androidTest/java/ch/epfl/culturequest/ProfileCreatorActivityTest.java

```diff
@@ -154,6 +162,24 @@ public void wrongUsernamesFailProfileCreation() throws InterruptedException {
         onView(withId(R.id.username)).check(matches(withHint(INCORRECT_USERNAME_FORMAT)));
     }
 
+    @Test
+    public void profilePictureButtonSendsPickerIntent() throws InterruptedException {
+        onView(withId(R.id.profile_picture)).perform(click());
+        intended(hasAction(Intent.ACTION_PICK));
+    }
+
+//    @Test
+//    public void afterPictureChosenGoToCrop() {
+//        Intent intent = new Intent();
+//        intent.setData(Uri.parse("content://media/external/images/media/1"));
+//        Instrumentation.ActivityResult result = new Instrumentation.ActivityResult(RESULT_OK, intent);
+//        intending(hasAction(Intent.ACTION_PICK)).respondWith(result);
+//
+//        onView(withId(R.id.profile_picture)).perform(click());
+//
+//        intended(hasComponent(UCropActivity.class.getName()));
+//    }
+
```

Comment:
Why is this test commented? Should he be removed?

Reply:
There is a problem with the permissions that needs to be granted on the CI. I will need to investigate this issue next sprint.

But in the meantime the tests are already written.

Reply:
Ok I see then you can keep it that way